parts/k8s/cloud-init/artifacts/setup-custom-search-domains.sh

```diff
@@ -2,9 +2,9 @@
 set -x
 source /opt/azure/containers/provision_source.sh
 
-sudo echo "  dns-search <searchDomainName>" >> /etc/network/interfaces.d/50-cloud-init.cfg
```

Comment:
Note that [SC2024](https://github.com/koalaman/shellcheck/wiki/SC2024) implies this was not working as expected. Maybe it wasn't an issue because the script is already run in a `root` context?